tests/testthat/test-sdf-stat.R

```diff
@@ -9,3 +9,45 @@ test_that("sdf_crosstab() works", {
   expect_setequal(names(df), c("cyl_gear", "3.0", "4.0", "5.0"))
   expect_setequal(df[, 1, drop = TRUE], c("8.0", "4.0", "6.0"))
 })
+
+test_that("sdf_quantile() works for a single column", {
+  sc <- testthat_spark_connection()
```

Comment:
```suggestion
  test_requires_version("2.0.0", "approxQuantile() is only supported in Spark 2.0.0 or above")

  sc <- testthat_spark_connection()
```

Reply:
I refactored the tests in 90f00e8, separating the single-column and multi-column tests so that the single-column version would run on all Spark versions, and it would only skip the multi-column (on Spark < 2). I don't think we need to skip when it's a single column, because that should work on all Spark versions.

Reply:
@wkdavis Looks like `test_requires(...)` is still needed here because the API method `sdf_quantile()` depends on is not present in Spark < 2

Aside from this everything else looks good.

Reply:
Anyhow, I guess I'll just take the liberty to commit this suggested change so that all tests will pass, and after that I'll be able to merge this PR.

Thanks for contributing to sparklyr!! 🙂

Reply:
@yitao-li That works for me, thank you!